internal/cephfs/volume.go

```diff
@@ -119,9 +119,17 @@ func (vo *volumeOptions) getSubVolumeInfo(ctx context.Context, volID volumeID) (
 	return &subvol, nil
 }
 
+type operationState int64
+
+const (
+	unknown operationState = iota
+	supported
+	unsupported
+)
+
 type localClusterState struct {
 	// set true if cluster supports resize functionality.
```

Comment:
This comment needs to be updated and can you also please change `resizeSupported` to `resizeState`. as it no more a Bool value.